_templates/skeleton/version/version.go.tmpl

```diff
@@ -8,43 +8,60 @@ import (
 	"github.com/gin-gonic/gin"
 )
 
-func New(c *gin.Context) (string, error) {
-	ver := ""
+const (
+	latestVersion = "-1"
+)
+
+type Version struct {
+	TargetVersion string
+}
+
+func NewVersion(c *gin.Context) (*Version, error) {
+	version := &Version{}
+
+	if err := version.initialize(c); err != nil {
+		return nil, err
+	}
+
+	return version, nil
+}
+
+func (self *Version) initialize(c *gin.Context) error {
```

Comment:
This method should extract out of `Version` structure, or include in `NewVersion` method.